packages/openactive-integration-tests/test/features/core/amending-order-quote/implemented/c2-with-different-details-test.js

```diff
@@ -91,7 +90,14 @@ FeatureHelper.describeFeature(module, {
 
   /** Fetch some new opportunities, amend the existing order with a C2 request, and then complete it */
   describe('Second Attempt - C2 -> B', () => {
-    const { state, flow } = getOpportunityFeedItemsWithNewState();
+    const state = new RequestState(logger, {
+      uuid,
+    });
+    const flow = new FlowHelper(state, {
+      stagesToSkip: new Set(['C1']),
```

Comment:
Interesting use of `Set()` here - is this pattern normal for config (not seen a library ask for this kind of thing in `options` type setups before?)

Reply:
Would it not be better to move the Set() into the constructor of FlowHelper?